src/index.html

```diff
@@ -8,100 +8,153 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
         <title>ESFiddle</title>
+        <script src="https://use.fontawesome.com/52284be701.js"></script>
```

Comment:
Actually, I did this in the head because it's what font awesome tells you to do, but it's asynchronous so it won't block the loading of the page

<img width="1208" alt="screen shot 2017-02-26 at 19 05 19" src="https://cloud.githubusercontent.com/assets/1051509/23342693/cff4ac30-fc56-11e6-83ca-6d69d033a457.png">